### PR TITLE
[DEV-3731] Fixes Add Prayer DateTime

### PR DESCRIPTION
**Pull request by cdanwards**
## DESCRIPTION

As a user, whenever I create a prayer, I want the datetime when it was created to be correct.

### What does this PR do, or why is it needed?

This PR brings in the moment-timezone package to enable us to correctly format the datetime whenever we create a prayer. Before we were using `new Date().toJSON()` and while this does correctly stringify the date for Rock, `toJSON()` actually uses `toISOString()` under the hood, which changes the date to UTC time. Because of this, date we were sending to Rock was 4 hours in the future.

I based my use of moment-timezone off of Vincent's use [here](https://github.com/ApollosProject/apollos-prototype/blob/bb05dc927e50f50f1e3da3e2ec988df19b982ada/packages/apollos-data-connector-rock/src/content-items/data-source.js#L164)

### How do I test this PR?

1) Start the App.
2) Navigate to Prayer.
3) Create a prayer.
4) Go to your prayers and make sure it displays the correct time.
5) If you really want to test, you can go to the [sql-command](https://alpha-rock.newspring.cc/page/147) in Rock and run the below command and look for your prayer and check the datetime.
```
SELECT *
FROM PrayerRequest
```

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by richarddubay — APPROVED** [review]
Looks like this works to me.

File: packages/apollos-data-prayer/src/prayer-requests/data-source.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import RockApolloDataSource from '@apollosproject/rock-apollo-data-source';
 import ApollosConfig from '@apollosproject/config';
+import moment from 'moment-timezone';
```
Comment (IsaacHardy) [inline packages/apollos-data-prayer/src/prayer-requests/data-source.js:3]: This isn't built into normal `moment`?

Comment (cdanwards) [reply packages/apollos-data-prayer/src/prayer-requests/data-source.js:3]: @IsaacHardy I tried it with the base moment, but it kept throwing an error. So, looking at their docs, it looks like `.tz` only imports from `moment-timezone`, but `moment` can import from either.

Comment (IsaacHardy) [reply packages/apollos-data-prayer/src/prayer-requests/data-source.js:3]: Sounds good.

**Pull request merged**